peribolos/knative.yaml

```diff
@@ -319,6 +319,14 @@ orgs:
         has_wiki: false
         squash_merge_commit_title: PR_TITLE
         squash_merge_commit_message: COMMIT_MESSAGES
+      tools:
+        allow_merge_commit: false
+        allow_rebase_merge: false
+        description: Tools used in Knative project
```

Comment:
Wrt the comments from @zroubalik and @dprotaso maybe just a bit more context in the description would help. 

E.g. "Devel(opment) and infra(structure) tools used in Knative". (shorter or longer version)

Otherwise I like the shorter `tools` name.

Reply:
As discussed in TOC, I'm changing the name to `toolbox`